core-web/libs/sdk/react/src/lib/components/BlockEditorRenderer/blocks/Contentlet.tsx

```diff
@@ -0,0 +1,65 @@
+import { ContentNode, CustomRenderer } from '../../../models/blocks.interface';
+
+// Replace this when we have a types lib
+export interface Contentlet {
+    hostName: string;
+    modDate: string;
+    publishDate: string;
+    title: string;
+    baseType: string;
+    inode: string;
+    archived: boolean;
+    ownerName: string;
+    host: string;
+    working: boolean;
+    locked: boolean;
+    stInode: string;
+    contentType: string;
+    live: boolean;
+    owner: string;
+    identifier: string;
+    publishUserName: string;
+    publishUser: string;
+    languageId: number;
+    creationDate: string;
+    url: string;
+    titleImage: string;
+    modUserName: string;
+    hasLiveVersion: boolean;
+    folder: string;
+    hasTitleImage: boolean;
+    sortOrder: number;
+    modUser: string;
+    __icon__: string;
+    contentTypeIcon: string;
+    variant: string;
+}
+
+type DotContentProps = ContentNode & {
+    customRenderers?: CustomRenderer;
+};
```

Comment:
Probably this too. All the types you use to create the blocks, should expose their props, so the dev can mimic the component with his own logic/template